### ISSUE TITLE
Training

### ISSUE THREAD

**Issue by SaiSurya9999** [hamuchiwa/AutoRCCar#151]
Does training stores all the video frames and action we took i mean forward or left or right.
Where will they be stored..??
Thank you..!!

**Comment by hamuchiwa** [comment]
Image along with label are saved only when there is a key press action. In the end, data is saved as a npz file in `training_data` directory.

**Comment by SaiSurya9999** [comment]
Thank you @hamuchiwa ..!!

**Comment by SaiSurya9999** [comment]
In the model_training folder we are having an npz file is it the data you have trained can we directly use model without training as the trained doc i mean npz  already present..?? (By creating the same environment as shown in the video)

**Comment by hamuchiwa** [comment]
There is no pre-trained model. If you refer to the data file in `model_train_test` folder, it's just a example to walk you through how to use neural network in opencv.

**Comment by SaiSurya9999** [comment]
ohh ok Thank you @hamuchiwa..!!

**Comment by Vaibhav18t** [comment]
Sir while trying to run the training model using collect_training_data.py it runs on the Anaconda prompt.
Then when I am trying to run the stream_client.py on the raspberry pi, the Anaconda running Server is getting terminated showing the following error. Please tell me how to clear this error. It is Urget. I hope to hear from you soon.
![capture](https://user-images.githubusercontent.com/39966602/53267961-c7098000-370a-11e9-8673-1e0a2f2b7680.PNG)

**Comment by hamuchiwa** [comment]
@Vaibhav18t Duplicated? #153

**Comment by Vaibhav18t** [comment]
Yes Sir. The code is the same. Made some changes in the IP address and port number, and as some changes in the car's arduino sketch for controls. Could you please tell me how to re-solve this issue??@hamuchiwa #153

**Comment by hamuchiwa** [comment]
@Vaibhav18t As mentioned in #153, find your arduino port on your windows computer, then replace the one in the code.

**Comment by Vaibhav18t** [comment]
Sir, which module should I use to connect my arduino to my pc wirelessly. Also I thank you for your help sir.
@hamuchiwa

**Comment by Vaibhav18t** [comment]
Sir how should I connect the arduino to my pc and at the same time to the car? Should I use a bluetooth module to connect to the arduino from my pc?

**Comment by hamuchiwa** [comment]
@Vaibhav18t Arduino is connected to the remote controller not the car, see #114.

**Comment by Vaibhav18t** [comment]
@hamuchiwa Sir we have made a motor car instead of the rc-car and we thought we would interface with the arduino with a bluetooth module with our pc. I want to ask you that if we give the bluetooth address instead of usb port address, will it work??
